gallery/src/components/NxFilterInput/NxFilterInputPage.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ .nx-filter-input-examples * {
+  box-sizing: border-box;
```

Comment:
Override the box-sizing to demonstrate the problem NxFilterInput has when the box-sizing is set incorrectly. This lets me verify the fix while also helping us identify the problem in the future if the box-sizing settings are messed with.

Reply:
> Override the box-sizing to demonstrate the problem NxFilterInput has when the box-sizing is set incorrectly.

Maybe include something about this as a code comment

Reply:
I'm not sure that I understand why we have this? We're demonstrating how to break the component?

Reply:
> > Override the box-sizing to demonstrate the problem NxFilterInput has when the box-sizing is set incorrectly.
> 
> Maybe include something about this as a code comment

I don't think this is necessary

Reply:
Also the indentation is screwed up

Reply:
Well, this demonstrates the component in a broken state when the styling is incorrect. Since we mentioned that we should expect components to work with `box-sizing: border-box` I needed a way to validate that the component worked correctly in that environment. I can either remove this style, or I could add a global style that affects all components in the gallery.